packages/bcer-api/app/src/migrations/1659555307942-FaqTable.ts

```diff
@@ -0,0 +1,14 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class FaqTable1659555307942 implements MigrationInterface {
+    name = 'NotesTable1659555306536'
```

Comment:
this needs to be updated as well to avoid duplicate name errors during migration!